packages/holodeck/server/index.js

```diff
@@ -5,54 +5,27 @@ import { Hono } from 'hono';
 import { cors } from 'hono/cors';
 import { HTTPException } from 'hono/http-exception';
 import { logger } from 'hono/logger';
-import { execSync } from 'node:child_process';
 import crypto from 'node:crypto';
 import fs from 'node:fs';
 import http2 from 'node:http2';
 import zlib from 'node:zlib';
-import { homedir, userInfo } from 'os';
 import path from 'path';
 
 /** @type {import('bun-types')} */
 const isBun = typeof Bun !== 'undefined';
 const DEBUG = process.env.DEBUG?.includes('holodeck') || process.env.DEBUG === '*';
 const CURRENT_FILE = new URL(import.meta.url).pathname;
 
-function getShellConfigFilePath() {
-  const shell = userInfo().shell;
-  switch (shell) {
-    case '/bin/zsh':
-      return path.join(homedir(), '.zshrc');
-    case '/bin/bash':
-      return path.join(homedir(), '.bashrc');
-    default:
-      throw Error(
-        `Unable to determine configuration file for shell: ${shell}. Manual SSL Cert Setup Required for Holodeck.`
-      );
-  }
-}
-
 function getCertInfo() {
   let CERT_PATH = process.env.HOLODECK_SSL_CERT_PATH;
   let KEY_PATH = process.env.HOLODECK_SSL_KEY_PATH;
-  const configFilePath = getShellConfigFilePath();
 
-  if (!CERT_PATH) {
-    CERT_PATH = path.join(homedir(), 'holodeck-localhost.pem');
-    process.env.HOLODECK_SSL_CERT_PATH = CERT_PATH;
-    execSync(`echo '\nexport HOLODECK_SSL_CERT_PATH="${CERT_PATH}"' >> ${configFilePath}`);
```

Comment:
IIRC the setup here exists for CI environments.

We should probably gate it with `process.env.CI` instead of removing it (this is the reason for the failing test scenarios in this PR), though there may be some other nuance that allows us to still remove this but make CI work.

I *think* the issue is that we don't re-source the ENV after running ensure-cert (see https://github.com/emberjs/data/blob/536192b15822c49226b462a905fa39e47a790fea/.github/actions/setup/action.yml#L75) so the paths aren't seen even though they are already present

Reply:
Yeah, I tried adding the source command to the ensure-cert script, but was unable to make it work, probably because the script is running on a child process.

Since the CI setup is running the ensure-cert script, I'll attempt to add the re-sourcing step to it.

Reply:
Added back the default values, only removing the bash file editing, so it should work for CI and locally as well.